hx_lti_initializer/views.py

```diff
@@ -369,7 +371,8 @@ def course_admin_hub(request):
             'debug': debug,
             'starter_object': starter_object,
             'starter_object_id': object_id,
-            'starter_collection_id': collection_id
+            'starter_collection_id': collection_id,
+            'utm_source': request.session.session_key if not request.session['is_staff'] else '',
```

Comment:
And yes, I was trying to leave it up to you to see if it was worth doing. I didn't know if sessions/cookies were a big deal on your end since you're all working within the same domain whereas we have one for the hxat and then edx.org. So I only made the changes where it would affect what Hx students saw not instructors. Since it seems like it is an issue for y'all as well would you like me to add it just where students will click or also everywhere instructors will click?